manifests/plugin/threshold.pp

```diff
@@ -2,12 +2,16 @@
 class collectd::plugin::threshold (
   $ensure   = 'present',
   $interval = undef,
+  Hash[String, Collectd::Threshold::Type]   $types   = {},
```

Comment:
can you replace `String` with `String[1]`?

Reply:
Not just here I guess :thinking: